Documentation/Registry.md

```diff
@@ -429,11 +433,32 @@ Otherwise, a server SHOULD respond with a status code of `404` (Not Found).
 HTTP/1.1 200 OK
 Content-Version: 1
 Content-Type: application/json
-Content-Length: 620
+Content-Length: 720
 Link: <https://packages.example.com/mona/LinkedList/1.1.1>; rel="latest-version",
       <https://packages.example.com/mona/LinkedList/1.0.0>; rel="predecessor-version"
+{
+  "id": "mona.LinkedList",
+  "version": "1.1.1",
+  "resources": [
+    {
+      "name": "source-archive",
+      "type": "application/zip",
+      "checksum": "a2ac54cf25fbc1ad0028f03f0aa4b96833b83bb05a14e510892bb27dea4dc812"
```

Comment:
Should this be `sha-256=a2ac54cf25fbc1ad0028f03f0aa4b96833b83bb05a14e510892bb27dea4dc812`?

Reply:
Now that we aren't using the standard `Digest` header, we don't need to specify an algorithm here. The `checksum` for a package is tied to the output of `swift package generate-checksum`, which is fixed to SHA256. I don't personally expect that we'll need to change this, but if we wanted to future-proof this, we could change this to:

```suggestion
      "checksum": {"sha-256": "a2ac54cf25fbc1ad0028f03f0aa4b96833b83bb05a14e510892bb27dea4dc812"}
```